addons/api/addon/handlers/cache-daemon-handler.js

```diff
@@ -167,7 +167,14 @@ export default class CacheDaemonHandler {
         // This isn't conventional but is better than returning an ArrayProxy
         // or EmberArray since the ember store query method asserts it has to be an array
         // so we can't just return an object.
-        records.meta = { totalItems: results?.length ?? 0 };
+        // Also include whether the results are incomplete or is still refreshing
+        // as part of the response
+        records.meta = {
```

Comment:
Like a P-R-O! nice!

Reply:
This isn't ideal IMO and I quite dislike it 😬, it's very hacky but I still don't have a better way of doing it unfortunately

Reply:
Not judjing at all, what I like it is in a few lines it is pretty clear what you ar defining and the origin of it, so I find it very descriptive an easy to understand